pyproject.toml

```diff
@@ -23,7 +23,7 @@ pylzma = "^0.5.0"
 PyYAML = "^5.1"
 arrow = "^0.15.7"
 colorama = "^0.4.1"
-gevent = "1.4.0"
+gevent = ">=20.6.0"
```

Comment:
this is not that simple,  https://github.com/threefoldtech/js-sdk/issues/1714#issuecomment-731488031

Reply:
> this is not that simple, [threefoldtech/js-sdk#1714 (comment)](https://github.com/threefoldtech/js-sdk/issues/1714#issuecomment-731488031)

I understand this. will work on it isa.